crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -1059,15 +1064,15 @@ fn is_airflow_builtin_or_provider(segments: &[&str], module: &str, symbol_suffix
 
 /// Returns `true` if the current statement hierarchy has a function that's decorated with
 /// `@airflow.decorators.task`.
-fn in_airflow_task_function(semantic: &SemanticModel) -> bool {
+fn in_airflow_task_function_def(semantic: &SemanticModel) -> bool {
     semantic
         .current_statements()
         .find_map(|stmt| stmt.as_function_def_stmt())
-        .is_some_and(|function_def| is_airflow_task(function_def, semantic))
+        .is_some_and(|function_def| is_airflow_task_function_def(function_def, semantic))
 }
 
 /// Returns `true` if the given function is decorated with `@airflow.decorators.task`.
-fn is_airflow_task(function_def: &StmtFunctionDef, semantic: &SemanticModel) -> bool {
+fn is_airflow_task_function_def(function_def: &StmtFunctionDef, semantic: &SemanticModel) -> bool {
```

Comment:
I think we should revert these two renames as it's very confusing - the only difference between the two is "in_" and "is_ prefix.

Reply:
(I know you're out this week, so will merge this and rename as I can't edit this PR.)

Reply:
Thanks for helping out! I tried to check whether I could allow modification, but it seems I do not have the permission either 🥲